test/common_quantization.py

```diff
@@ -104,6 +112,13 @@ def checkQuantizedLinear(self, mod):
         self.assertEqual(type(mod), nnq.Linear)
         self.assertEqual(mod.bias.dtype, torch.qint32)
 
+    def checkDynamicQuantizedLinear(self, mod):
```

Comment:
Isn't this check too specific to be put in the common file? If you really need to factor something out in the test, but the method is too specific, you can always create a function in the child test class that you are creating, instead of the common parent class.

Reply:
this looks fine, it's checking on the leaf linear module.